cmd/agent/app/options/options.go

```diff
@@ -80,6 +81,7 @@ func (o *GrpcProxyAgentOptions) Flags() *pflag.FlagSet {
 	flags.StringVar(&o.ProxyServerHost, "proxy-server-host", o.ProxyServerHost, "The hostname to use to connect to the proxy-server.")
 	flags.IntVar(&o.ProxyServerPort, "proxy-server-port", o.ProxyServerPort, "The port the proxy server is listening on.")
 	flags.StringSliceVar(&o.AlpnProtos, "alpn-proto", o.AlpnProtos, "Additional ALPN protocols to be presented when connecting to the server. Useful to distinguish between network proxy and apiserver connections that share the same destination address.")
+	flags.StringVar(&o.HealthServerHost, "health-server-host", o.HealthServerHost, "The host address to listen on, without port.")
```

Comment:
Any new configuration options should be reflected in the Print() method below.

Reply:
Would also be nice if we could validate the value configured, though that seems harder than it sounds (and not required .. yet).

Reply:
Can we add the HealthServerHost field to the print() method?

Reply:
Added it, sorry, this PR fell off my plate during the end of last year